model/src/test.rs

```diff
@@ -182,7 +187,7 @@ impl Test {
                         Outcome::Pass => TestUserState::Passed,
                         Outcome::Fail => TestUserState::Failed,
                         Outcome::Timeout => TestUserState::Failed,
-                        Outcome::Unknown => {
+                        Outcome::Unknown | Outcome::InProgress => {
```

Comment:
I don't remember we were returning "Passed" for a test with Outcome::Unknown, do you? Seems a little weird to say an InProgress test has "Passed", but I don't know the consequences or how that might be encountered.

Reply:
I thing the original logic was that we check to make sure there are no failed tests.